### ISSUE TITLE
Issuing Memes is a bit confusing at first

### ISSUE THREAD

**Issue by Bradymck** [district0x/memefactory#133]
When I first had a meme added to the registry, it was unclear I could or should do something after that. This would be a good place for a prompt/notification as mentioned in #129 . Something taking you to your Meme Folio to issue and list memes would be great. 

Also after issuing, it was not the most intuitive experience to click your meme to sell it. A prompt or notification would be good to have here as well. 

Issuing and selling should all be done on the back of a card as one action, or in same "back of the card" panel at least showing that it will be two transactions. Doing two transactions in two places was a bit awkward without being told there would be two. 

Even without the above suggestions, after issuing your memes it could say something like: 

**'Congratulations your card has been issued on Ethereum! You can now click your card to list it on the marketplace!'** 

This would eliminate the need to move issuing to the back of the card.

**Comment by madvas** [comment]
I agree, there should be more information given to the user about the process. But we need to keep this as 2 transactions, due to larger gas requirements for each of the transaction.

**Comment by Bradymck** [comment]
Cool I'll leave this open for now.

**Comment by Bradymck** [comment]
More technical info in email notifications when meme makes it into registry. Remind people to issue and remind people to list meme on market. A poke if they forget to, or don't do it.

**Comment by madvas** [comment]
Implement email notification that notifies creator when his meme got into the registry. Email will contain meme image and following text: 
Subject: "[meme-title] just got into the registry!"
Headline: "[meme-title] just got into the registry!"
Text: "Your meme [meme-title] has successfully got into Meme Factory registry. Now you can mint cards and start selling!" 
Button text: "Mint Cards"
Button links to My Memefolio - Created
